src/Essentials/src/Share/Share.shared.cs

```diff
@@ -53,11 +60,23 @@ public static Task RequestAsync(ShareMultipleFilesRequest request)
 			if (request.Files.Any(file => file == null))
 				throw new ArgumentException(FileNullExeption(nameof(request.Files)));
 
-			return PlatformRequestAsync(request);
+			return Current.RequestAsync(request);
 		}
 
 		static string FileNullExeption(string file)
 			=> $"The {file} parameter in the request files is invalid";
+
+#nullable enable
```

Comment:
Scoping `#nullable` to the entire file leads to non-trivial API questions around `ShareMultipleFilesRequest`.